master/docs/relnotes/0.9.0b7.rst

```diff
@@ -0,0 +1,75 @@
+Release Notes for Buildbot 0.9.0b6
+==================================
+
+
+The following are the release notes for Buildbot 0.9.0b7
+This version was released on February 14, 2016.
+
+See :ref:`Upgrading to Nine` for a guide to upgrading from 0.8.x to 0.9.x
+
+Master
+------
+
+Features
+~~~~~~~~
+
+Fixes
+~~~~~
+
+* Fix incompatibility with MySQL-5.7 (:bug:`3421`)
+* Fix incompatibility with postgresql driver psycopg2 (:bug:`3419`, further regressions will be catched by travis)
+* Fix regressions in forcescheduler UI (:bug:`3416`, :bug:`3418`, :bug:`3422`)
```

Comment:
Ahh.. Just fired it back up. And yes it's fixed now. Sorry I expected the pin to do something different.